src/content/docs/en/extend/add-content-collections.mdx

```diff
@@ -0,0 +1,326 @@
+---
+title: Tutorial - Extend with Content Collections
+description: >-
+  Convert the Build a Blog tutorial code from file-based routing to content collections.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+import Since from '~/components/Since.astro'
+import TypeScriptSettingTabs from '~/components/tabs/TypeScriptSettingTabs.astro'
+import RecipeLinks from "~/components/RecipeLinks.astro"
+
+
+**Content collections** are a powerful way to manage groups of similar content, such as blog posts. Collections help to organize your documents, validate your YAML frontmatter, and provide automatic TypeScript type-safety for all of your content (even if you don't write any TypeScript yourself).
+
+## Prerequisites
+
+You will need **an existing Astro project with Markdown or MDX files in the `src/pages/` folder**.
+
+This tutorial uses the [Build a Blog tutorial's finished project](https://github.com/withastro/blog-tutorial-demo) to demonstrate converting a blog to content collections. You can fork and use that codebase locally, or complete the tutorial in the browser by [editing the blog tutorial's code in StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/complete?file=src%2Fpages%2Findex.astro).
+
+You can instead follow similar steps in your own Astro project, but you will need to adjust the instructions for your own codebase. We recommend using our sample project to complete this short tutorial first. Then, you can use what you have learned to work on your own codebase.
```

Comment:
This paragraph felt very dense, maybe reducing a bit of the "own"s here helps, but also maybe we can find a way to structure it without repeating "your codebase" as well
```suggestion
You can instead follow similar steps in your own Astro project, but you will need to adjust the instructions according to your codebase. We recommend using our sample project to complete this short tutorial first. Then, you can use what you have learned to work on your codebase.
```

Reply:
'I adjusted your suggestion so I could commit it. If it's dense, then I'll make it 2 diff paragraphs, and I'll adjust wording for easier readability and flow! Thanks Yan!